maps_dashboards/public/components/layer_control_panel/layer_control_panel.tsx

```diff
@@ -34,46 +37,49 @@ interface MaplibreRef {
 
 interface Props {
   maplibreRef: MaplibreRef;
+  mapIdFromUrl: string;
+  setLayers: (layers: ILayerConfig[]) => void;
+  layers: ILayerConfig[];
 }
 
-const LayerControlPanel = ({ maplibreRef }: Props) => {
+const LayerControlPanel = memo(({ maplibreRef, mapIdFromUrl, setLayers, layers }: Props) => {
```

Comment:
Whats the intention behind memoizing this component? Did you observe a performance issue?

Reply:
Makes sense :) thanks